### ISSUE TITLE
Poll: listener called according to number of pages are opened

### ISSUE THREAD

**Issue by iamsiddiquek** [primefaces/primefaces#5106]
## 1) Environment
- PrimeFaces version: 6.2 and 7.1
- Does it work on the newest released PrimeFaces version ?
No with the current version which is deployed on-site also has this issue.
 Version? :   7.1
- Application server + version: Wild-fly 8.2 
- Affected browsers: Chrome Version 76.0.3809.132 (Official Build) (64-bit)

## 2) Expected behaviour:

p:poll should wait and call only once after the interval time even though how many pages duplicate pages are open.
...

## 3) Actual behaviour
whenever I am opening multiple duplicates pages it is increasing count according to the number of pages I have opened after interval time and listener calls.
..

## 4) Steps to reproduce
Open multiple or 5 duplicates pages
[Primefaces  Poll](https://www.primefaces.org/showcase/ui/ajax/poll.xhtml)

and after interval time it is increasing value by pages.
.. 

## 5) Sample XHTML

`<h:form>
    <h:outputText id="txt_count" value="#{counterView.number}" />
    <p:poll interval="1" listener="#{counterView.increment}" update="txt_count" />
</h:form>`
..

## 6) Sample bean
`@ManagedBean
@ViewScoped
public class CounterView implements Serializable {
     
    private int number;
 
    public int getNumber() {
        return number;
    }
 
    public void increment() {
        number++;
    }
}`
..

**Comment by tandraschko** [comment]
So you are looking for a poll, which interval synchronizes for all open tabs for the same view?

**Comment by tandraschko** [comment]
I don't know your usecase but poll is currently by design and a multiWindowSupport is hard to implement. It will likely be a won't fix.

Can you post your usecase?

**Comment by iamsiddiquek** [comment]
I had a requirement to block a user for some period of time and show it into the main page how much time is remaining and after a certain time unblock a user and for this feature, I used the Poll. 
and there was no feature synchronizes interval value.
I was opening multiple duplicates pages and according to a number of pages, the p:poll was listener called on that listener was 
> counter--;

means it was deleting one value at one call and when multiple calls it was removing multiple values/.

**Comment by melloware** [comment]
@christophs78 @tandraschko I wonder if this could be implemented now that MultiViewState can be added to any component and the new Window handling code was added.

**Comment by tandraschko** [comment]
There is no relation to multiViewState.
I think the topic is to special and the User should handle it in the Controller and make a own session scoped state.

**Comment by melloware** [comment]
Agreed.  I am going to close this ticket for now in the name of cleaning up this issues board.  As there is nothing we can really do about it and a custom solution is warranted.
